Source/WebCore/platform/graphics/ca/cocoa/PlatformCALayerCocoa.h

```diff
@@ -26,6 +26,7 @@
 #pragma once
 
 #include "PlatformCALayer.h"
+#include "PlatformCALayerDelegatedContents.h"
```

Comment:
Is this include required if the rest of the changes to header doesn't use it?

Reply:
You're right, it's not needed. Removed.